### DISCUSSION TITLE
Crash Android app – java.lang.NoSuchMethodError: No virtual method queryPurchase

### DISCUSSION THREAD

**Discussion by whyamsx** [hyochan/react-native-iap#1956] [Q&A]
I followed all the steps to install the module, but when I try to open my application, it immediately closes with an error:
`
java.lang.NoSuchMethodError: No virtual method queryPurchases(Ljava/lang/String;)Lcom/android/billingclient/api/Purchase$PurchasesResult; in class Lcom/android/billingclient/api/BillingClient; or its super classes (declaration of 'com.android.billingclient.api.BillingClient' appears in /data/app/~~FoR8L1AMXFFXmbdRD_fM2g==/com.myapp-xM5XOFzRKtJr5CCRKb25hA==/base.apk)
`

I tried integration for AndroidX, also without it - all to no avail.

**Comment by andresesfm** [comment]
Please open a ticket using the template adding all the details
